LICENSE

```diff
@@ -0,0 +1,25 @@
+BSD 2-Clause License
+
+Copyright (c) 2013-2019, Daniel Thomas, Daniel Carter, Alastair Beresford, Andrew Rice, Daniel Wagner and Sergio Pastrana
```

Comment:
I don't think Daniel Wagner wrote any of the lines in this repository as he had left before we started this part of the project. Missing an Oxford comma. I don't think anyone else is missing as vulnerability submissions are acknowledged separately.